js_modules/dagster-ui/packages/ui-core/src/sensors/SensorDetails.tsx

```diff
@@ -79,6 +80,11 @@ export const SensorDetails = ({
   } = usePermissionsForLocation(repoAddress.location);
   const {canUpdateSensorCursor} = permissions;
 
+  const alertDetails = useSensorAlertDetails({
```

Comment:
any reason for this to be a hook instead of a component?

Reply:
Ah yeah, these could be components. I used a hook for the Jobs version (since it returns a tab config) and just went ahead and did the same for these, but yeah, I didn't need to.